fluidBuild.config.cjs

```diff
@@ -165,6 +165,7 @@ module.exports = {
 			"docs/themes/thxvscode/layouts/",
 			"docs/themes/thxvscode/static/assets/",
 			"docs/tutorials/.*\\.tsx?",
+			"packages/common/client-utils/src/cjs/package.json",
```

Comment:
nit: description as to why this is added :)

Reply:
I think up mean a comment in this file. The PR (commit) description does mention this file. I will follow on with an update for this. Good call.

Reply:
oops...I meant a comment.

Reply:
#19895